code/game/machinery/doors/door.dm

```diff
@@ -325,11 +326,14 @@
 	operating = FALSE
 	air_update_turf(1)
 	update_freelook_sight()
-	if(safe)
+
+	if(!can_crush)
+		return TRUE
+	else if(safe)

```

Comment:
No need for the else if you are returning early

Reply:
that is true.